app/cdap/components/ConnectionList/utils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export const joinStringsWithSpaceSeparator = (stringsData = []) => stringsData.join(' ');
```

Comment:
there is no need for this function - its easier and more explicit to just us Array.prototype.join

Reply:
Done.